opencti-platform/opencti-graphql/src/modules/notifier/notifier-domain.ts

```diff
@@ -55,6 +55,12 @@ export const notifierGet = (context: AuthContext, user: AuthUser, notifierId: st
 };
 
 export const notifierEdit = async (context: AuthContext, user: AuthUser, notifierId: string, input: EditInput[]) => {
+  const fieldsToValidate = {
+    name: '',
```

Comment:
Why do you need the name property ?

Reply:
it's for types reasons: validateNotifier input should have a 'name'

Reply:
I am not really comfortable to add some unused property to satisfy a method contract.
It's better to change the method contract by saying
`validateNotifier = (notifier: {notifier_connector_id: string, notifier_configuration: string} )`

Reply:
right! fixed :)